suzieq/poller/writers/parquet.py

```diff
@@ -41,12 +44,12 @@ def __init__(self, **kwargs):
         self.root_output_dir = output_dir
 
     def write_data(self, data: Dict):
-        """Write the data into the Parquet output directory"
+        """Write the data into the Parquet output directory
 
         Args:
             data (Dict): dictionary containing the data to store.
         """
-        cdir = f"{self.root_output_dir}/{data['topic']}/"
+        cdir = os.path.join(self.root_output_dir, data['topic'])
```

Comment:
Not that it matters, but f-strings are more performant than os.path.join I presume, and isn't python3 recommending the use of Pathlib? Again, just curious, does not affect approval whatever you choose.